src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,58 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+interface Props {
+  tabs: Tab[],
+  selectedTabId: string,
+  onActiveTab: (tab: Tab) => void,
+  selectedTab: Tab,
+}
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onActiveTab,
+  selectedTab,
```

Comment:
Better calculate `selectedTab` according to id in this component.

Reply:
This is code duplication. Why if you can use it from the App.